src/Groups/matrices/MatGrp.jl

```diff
@@ -160,18 +160,19 @@ end
 group_element(G::MatrixGroup, x::GapObj) = MatrixGroupElem(G,x)
 
 function assign_from_description(G::MatrixGroup)
-   if G.descr==:GL G.X=GAP.Globals.GL(G.deg,codomain(G.ring_iso))
-   elseif G.descr==:SL G.X=GAP.Globals.SL(G.deg,codomain(G.ring_iso))
-   elseif G.descr==:Sp G.X=GAP.Globals.Sp(G.deg,codomain(G.ring_iso))
-   elseif G.descr==Symbol("GO+") G.X=GAP.Globals.GO(1,G.deg,codomain(G.ring_iso))
-   elseif G.descr==Symbol("SO+") G.X=GAP.Globals.SO(1,G.deg,codomain(G.ring_iso))
+   F = codomain(G.ring_iso)
+   if G.descr==:GL G.X=GAP.Globals.GL(G.deg, F)
+   elseif G.descr==:SL G.X=GAP.Globals.SL(G.deg, F)
+   elseif G.descr==:Sp G.X=GAP.Globals.Sp(G.deg, F)
+   elseif G.descr==Symbol("GO+") G.X=GAP.Globals.GO(1, G.deg, F)
+   elseif G.descr==Symbol("SO+") G.X=GAP.Globals.SO(1, G.deg, F)
    elseif G.descr==Symbol("Omega+")
       # FIXME/TODO: Work around GAP issue <https://github.com/gap-system/gap/issues/500>
       # using the following inefficient code. In the future, we should use appropriate
       # generators for Omega (e.g. by applying a form change matrix to the Omega
       # generators returned by GAP).
       # This also compensates for the fact that Omega(-1,2,q) is not supported in GAP.
-      L = GAP.Globals.SubgroupsOfIndexTwo(GAP.Globals.SO(1,G.deg,codomain(G.ring_iso)))
+      L = GAP.Globals.SubgroupsOfIndexTwo(GAP.Globals.SO(1, G.deg, F))
```

Comment:
I guess once a new forms release is in the package distribution and shipped with GAP.jl, we can also replace this code, right?

Reply:
Yes.
(In principle one could upgrade the Forms package using the functions from `GAP.Packages`, but this would be confusing.)